src/services/Content/ContentQueries.ts

```diff
@@ -222,7 +225,7 @@ export const useFetchGpgKey = () => {
       const description = typeof data === 'string' ? data : data?.message;
       notify({
         variant: AlertVariant.danger,
-        title: 'Error fetching gpg key from provided url',
+        title: 'Error fetching GPG key from provided url',
```

Comment:
```suggestion
        title: 'Error fetching GPG key from provided URL',
```

Is this user visible text? If so, use "URL" for consistency and conformity with  style guides.